tests/test_plot.py

```diff
@@ -180,7 +181,8 @@ def test_make_2d_axes_inputs_outputs():
 
     # Check gridspec kwargs
     fig, axes = make_1d_axes(paramnames_x, gridspec_kw=dict(wspace=0.1))
-    ws = axes[0].get_subplotspec().get_gridspec().get_subplot_params().wspace
+    ws = (axes.iloc[0].get_subplotspec()
+          .get_gridspec().get_subplot_params().wspace)
```

Comment:
Ugly line break, how about this?

```python
    ws = axes['A'].get_subplotspec().get_gridspec().get_subplot_params().wspace
```

Reply:
It's too long to get through flake8. I didn't want to put it in!

Reply:
sorry I've just read this properly - sure